### DISCUSSION TITLE
locking touch from buds

### DISCUSSION THREAD

**Discussion by avstoyanov** [timschneeb/GalaxyBudsClient#118] [General]
Hello, I was wondering if it is possible to somehow toggle the touch lock from the earbuds themselves. Since the touchscreen would obviously be locked, I am not sure how that would work. Is it possible to detect a triple tap on the edge?
Accidentally tapping the earbuds while adjusting or putting them on/taking them off is probably the biggest downfall of the galaxy buds+ in my opinion.

**Comment by timschneeb** [comment]
> Hello, I was wondering if it is possible to somehow toggle the touch lock from the earbuds themselves. Since the touchscreen would obviously be locked, I am not sure how that would work. 

I could add a custom touch action for locking the touchpads, but normally once you activated the lock, it is impossible to unlock them again without using the Android or Windows app. 
But I found a possible workaround to detect touch presses even when the pads are locked; however, I can only test it for the Buds+ since the detection method is different for every Galaxy Buds model.
I'll start testing whether my idea would actually work soon, but no guarantees yet. Also, this would only work on Windows/Linux systems with this app set up; not portable to Android and other devices.

> Is it possible to detect a triple tap on the edge?

No, not possible unless Samsung decides to implement this in their firmware.
